driver/modern_bpf/programs/attached/events/sched_process_exec.bpf.c

```diff
@@ -175,9 +175,12 @@ int BPF_PROG(t1_sched_p_exec,
 	auxmap__store_s32_param(auxmap, (s32)loginuid);
 
 	/* Parameter 20: flags (type: PT_FLAGS32) */
-	/// TODO: we still have to manage `exe_writable` flag.
 	u32 flags = 0;
 	struct inode *exe_inode = extract__exe_inode_from_task(task);
+	if(extract__exe_writable(task, exe_inode))
```

Comment:
The verifier seems able to handle all the complexity in just one BPF prog, but maybe in the future, we will need to split this prog since this function is complex. Right now I would keep it as it is since it works, we are always in time to split it in a second step